{{cookiecutter.project_dirname}}/{{cookiecutter.django_settings_dirname}}/urls_apibrowser.py

```diff
@@ -0,0 +1,36 @@
+"""API browser URL configuration."""
+
+from django.contrib.admin.views.decorators import staff_member_required
+from django.urls import include, path
+from drf_spectacular.views import SpectacularAPIView, SpectacularSwaggerView
+
+urlpatterns = [
+    path(
+        "browser/",
+        include(
+            (
+                [
+                    path(
+                        "schema/",
+                        staff_member_required(
```

Comment:
instead using `staff_member_required` here, drf_spectacular has a generic settings to protect his endpoints:
```
SPECTACULAR_SETTINGS = {
     "SERVE_PERMISSIONS": ["rest_framework.permissions.IsAuthenticated"],`
     ...
}
```